### ISSUE TITLE
Contact info for enhancement requests

### ISSUE THREAD

**Issue by TAZ427** [culmor30/cgeo-wear#11]
Hello, what's the best way to contact you for enhancement requests.

A couple idea's that came to mind were the ability to pull the hint in and also waypoints.

I think both would be potential time savers for cachers.

Best Regards,
TAZ427

**Comment by culmor30** [comment]
You can post enhancement requests on this issue tracker, or email me at JavaDog.Dev@gmail.com.

Waypoints are already compatible, though you have to manually select the waypoint in c:geo then click Navigate > Android Wear as you would with a regular cache.

**Comment by TAZ427** [comment]
Thanks, I'm really looking forward to using this.  I found out about it last night and ordered an Asus ZenWatch today.  I'm one of the hosts on geocaching podcast (geocachingpodcast.com) and one of my fellow hosts had brought up the app at the end of the show last night.  We had been chatting a few weeks ago and I mentioned that this is the one feature I'm waiting for to drive me to buy a smart watch. 

I've done a fair amount of embedded processing development, but not for smartphones.  And I've only used github for some code sharing on 3D printer development.  

I may just have a deeper look and see if there's anyway I can contribute, other than with enhancement requests.
